symplyphysics/laws/hydro/volume_of_the_submerged_body_part_from_body_density_and_liquid_density.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## If the body is on the surface of a liquid (floating), then only two forces act on it (Archimedes up and gravity down), which balance each other.
+
+## Law: V_sub / V = ⍴ / ⍴_liquid
+## Where:
+## V_sub is the volume of the submerged body part
+## V is full body volume
+## ⍴ is body density
+## ⍴_liquid is the density of the liquid
+
+## Conditions
+## The body is immersed in only one liquid.
+## Gravity should be equal to the force of Archimedes.

```

Comment:
This condition is better explained as the body should float, so the forces are in equilibrium.